### PR TITLE
Added a step by step install guide

### PR DESCRIPTION
**Pull request by juliendorra**
should be very useful for people with older install and/or not used to the terminal.
Also everything in one place :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,4 +23,71 @@ A build using an optical switch can be found at http://www.thingiverse.com/thing
...
 
-WARNING: I am **not responsible** for any failed prints. Use at your own risk.
\ No newline at end of file
+WARNING: I am **not responsible** for any failed prints. Use at your own risk.
+
+=== How to install ===
+
+For some older Octopi installs, the plugin won't work before you make sure you have everything set up to date. Here are the steps to ensure you have everyting in place for the plugin.
+ 0- you can check your system version with 
+- `uname -a
```
Comment (jonaslorander) [inline README.md:32]: You seem to be missing a ` at the end of every command.

Comment (Actpohomoc) [reply README.md:32]: Very nice guide. Many thanks. I've used your version to setup the plugin.

**Review by MoonshineSG — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (juliendorra) [reply README.md:32]: @jonaslorander you're right, this should be corrected
@Actpohomoc thanks, happy that it helped you!

**Comment by juliendorra** [comment]
@MoonshineSG @jonaslorander Corrective pullrequest for the backtick typos of this one : https://github.com/MoonshineSG/Octoprint-Filament/pull/21 
:-(

**Comment by Actpohomoc** [comment]
I did the sensor combined with filament arm on my PrintrBot Simple Metal
and start use this plugin. Many thanks. Now have another issue. In case
filament running out pause will be very good to decrease the temperature of
hot end. Otherwise it is not good and all rest of the filament leaks out.

1 трав. 2017 р. 12:27 пп "Julien Dorra" <notifications@github.com> пише:

**Comment by MoonshineSG** [comment]
use the `devel` version which triggers events which you can then use to trigger your on favourite action...
of course you need to reheat the head to the old temperature once you change the filament...

**Comment by Actpohomoc** [comment]
Thanks. For the moment it is difficult to me. I just wanted to share what
is important to me now when I start to use the filament running out sensor.
I think the increasing to the target temperature has to be inside the
unpause function. Printer should wait until temperature rich the target
value.
What do you think?


1 трав. 2017 р. 3:28 пп "ǝuıɥsuooɯ" <notifications@github.com> пише:

**Comment by MoonshineSG** [comment]
thats why the devel version doesnt enforce "pause" but allows the user the customise what gcode to be executed on runout and reload. If more complex behaviour is required - like temperature set on either of the events - the triggered events can be used by other plugins to play with.

as the plugin is open source, another option is to "butcher it" and get exactly what you need from it 

to be honest, I wrote this plugin as a coding example to someone and it has grown into what it is today due to the interest from the community.  fun fact: I do not use a filament sensor, or this plugin :D

File: README.md (hunk 1 above)
Comment (jonaslorander) [reply README.md:32]: Looks good now, and thanks for the guide! :)

**Pull request merged**